cli/cli.go

```diff
@@ -160,7 +160,7 @@ func (c *RootCLI) loadLog(cc *cobra.Command) (func(), error) {
 		os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0644)
 	if err != nil {
 		log.Error().Err(err).Msg("SetLog.OpenFile.Fail")
-		return func() {}, err
```

Comment:
The functions like this can't be caught by local go test but can be caught by codecov.